docs/monitoring.md

```diff
@@ -116,3 +116,127 @@ A **Data Collection Endpoint** provides a set of Azure-hosted endpoints that acc
 
 - Only **metrics** are sent to the DCE.
 - The **metrics ingestion endpoint** on the DCE acts as the **remote write target** for the Prometheus server running in the AKS cluster.
+
+## Azure Front Door Monitoring
+
+Azure Front Door metrics and logs are available in Grafana through two complementary approaches:
+
+### 1. Direct Azure Monitor Metrics (No Configuration Required)
+
+Azure Front Door automatically publishes platform metrics to Azure Monitor. These metrics are immediately available in Grafana without any additional configuration:
+
+**Available Metrics:**
+- Request count and rate
+- Latency (backend, total)
+- Cache hit ratio
+- Error rates (4xx, 5xx)
+- Backend health percentage
+- Web Application Firewall (WAF) request count
+
+**How to Query in Grafana:**
+1. Add **Azure Monitor** as a data source in Grafana (typically pre-configured)
+2. Create a new dashboard panel
+3. Select Azure Monitor data source
+4. Choose:
+   - **Subscription**: Your Azure subscription
+   - **Resource Group**: `global`
+   - **Resource Type**: `Microsoft.Cdn/profiles`
+   - **Resource**: Your Front Door profile name (e.g., `arohcpdev`)
+   - **Metric Namespace**: `Microsoft.Cdn/profiles`
+   - **Metric**: Select from available metrics (e.g., `RequestCount`, `TotalLatency`, `Percentage4XX`)
+
+**Advantages:**
+- Zero configuration required
+- Real-time metrics
+- Standard Azure Monitor aggregations (avg, min, max, sum, count)
+
+**Limitations:**
+- Metrics only (no detailed logs)
+- Limited retention (90 days by default)
```

Comment:
Will the logs be stored only for 90 days? We should discuss with the team on the data retention policy.